tests/chocolatey-tests/chocolatey.Tests.ps1

```diff
@@ -257,6 +257,21 @@ exit $error.count
         }
     }
 
+    Context 'Licensed features error on FOSS (<_> command)' -ForEach @( 'upgrade', 'install') -Tag FossOnly {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot
+            $Output = Invoke-Choco $_ installpackage --installdirectory="'C:\Temp'"
+        }
+
+        It "Should exit with Failure (1)" {
+            $Output.ExitCode | Should -Be 1 -Because $Output.String
+        }
+
+        It "Should report that is a licensed only option" {
+            $Output.Lines | Should -Contain "It appears you are attempting to use options that may be only available in licensed versions of Chocolatey ('--installdirectory='c:\temp''). There may be ways in the open source edition to achieve what you are looking to do. Please remove the argument and consult the documentation." -Because $Output.String
+        }
+    }
+

```

Comment:
Since the change about throwing the exception was backed out, this test is no longer valid.

Reply:
I've removed this addition.